src/theme/ItaliaTheme/Components/_mobileMenu.scss

```diff
@@ -105,7 +105,9 @@
 
               margin-left: 0;
 
-              .icon {
+              svg.icon {
+                width: 1em;
+                fill: currentColor;
                 width: 28px;
```

Comment:
Occhio che c'è width due volte quindi il tuo nuovo `1em` è ignorato così

Reply:
In realtà mi sono dimenticata di eliminare quell'`1em`, lo stile che veniva applicato prima di spostare questi stili nel file giusto, era 28x28px, correggo